usage/README.rst

```diff
@@ -72,22 +72,24 @@ standard output (caught here with nodehour_elapsed.out).  Below is an
 explanation of all of the possible options to use with usage.py.
 
 **-u** list
-
     specifies user(s) to gather data for (ex: -u 'mxk,thrush'); conflicts with
     **-j**.
 
 **-j** list 
-
     specifies job id(s) to gather data for(ex: -j '108408,108425,108426'); 
     conflicts with **-u**
 
-**-t** string
+**-f** list
+    specifies job id(s) that have failed that should still be included in the
+    plot(ex: -f '125653,125654').  Can be used with both **-u** or **-j**.
+    This argument is only needed if there are failed jobs in the list of jobs
+    to be plotted. 
```

Comment:
This description is not entirely accurate. Current implementation allows only to use `-f` option only with `-j`.